frontend/src/components/common/EmptyState.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Center, Text, VStack } from "@chakra-ui/react";
+
+import { SadFaceIcon } from "../../assets/icons";
+
+interface EmptyStateProps {
+  items: string;
+}
+const EmptyState = ({ items }: EmptyStateProps): React.ReactElement => (
+  <Center
+    borderRadius="lg"
+    borderWidth="1px"
+    flex="1"
+    padding={20}
+    width="100%"
```

Comment:
I think the design specs for the border are a bit different, could you update your implementation to match the design? i.e colour should be `#DFDFDF` and border radius should be `0`

Reply:
Done!